pmlc/rt/vulkan/wrappers.cc

```diff
@@ -28,10 +28,28 @@ static llvm::ManagedStatic<VulkanRuntimeManager> vkRuntimeManager;
 
 class VulkanRuntimeManager {
 public:
-  VulkanRuntimeManager() = default;
+  VulkanRuntimeManager() {
+    int64_t buffer_size = 9;
+    value[0] = reinterpret_cast<float *>(malloc(buffer_size * sizeof(float)));
+    value[1] = reinterpret_cast<float *>(malloc(buffer_size * sizeof(float)));
+    value[2] = reinterpret_cast<float *>(malloc(buffer_size * sizeof(float)));
+    for (int i = 0; i < 9; i++) {
+      *(value[0] + i) = i + 1;
+      *(value[1] + i) = i + 1;
+      *(value[2] + i) = 0;
+    }
+  }
```

Comment:
This might be for testing, but it can't remain here long term.

Reply:
Yes, I was using this for test.

Reply:
I will delete this .